setup.cfg

```diff
@@ -15,6 +15,7 @@ replace = __version__ = "{new_version}"
 universal = 1
 
 [flake8]
+ignore = E203
```

Comment:
Thanks for linking the documentation but it's not needed if the code gets simplified

```suggestion
```